packages/turbo-gen/src/commands/run/prompts.ts

```diff
@@ -47,26 +47,26 @@ export async function customGenerators({
 }
 
 export async function chooseGeneratorTemplate() {
-  return await inquirer.prompt<{ answer: "ts" | "js" }>({
```

Comment:
why do the awaits go away?

Reply:
They're redundant here since we don't need the result in this function, we can just return the promise and let the caller handle it. 

`const foo = async () => await somethingAsync()` 
is the same as
`const foo = async () => somethingAsync()`

Either way the caller will have to:
`const bar = await foo()`